### PR TITLE
Font remove noarch

### PR DESCRIPTION
**Pull request by fosslinux**
Mass remove of noarch for these font packages, which seem to be changed quite rarely.

### CODE REVIEW

File: srcpkgs/font-adobe-source-code-pro/template [hunk 1]
```diff
@@ -1,8 +1,7 @@
 # Template file for 'font-adobe-source-code-pro'
 pkgname=font-adobe-source-code-pro
 version=2.030R
-revision=2
-archs=noarch
+revision=6
```
Comment (Chocimier) [inline srcpkgs/font-adobe-source-code-pro/template:4]: =3

File: srcpkgs/font-fira-otf/template [hunk 2]
```diff
@@ -1,8 +1,7 @@
 # Template file for 'font-fira-otf'
 pkgname=font-fira-otf
 version=4.202
-revision=3
-archs=noarch
+revision=7
```
Comment (Chocimier) [inline srcpkgs/font-fira-otf/template:4]: =4

**Review by Chocimier — COMMENTED** [review]
Going to check if they build.

**Comment by fosslinux** [comment]
They all built for me. I will check for any more weird revbumps.

File: srcpkgs/font-libertine-ttf/template [hunk 3]
```diff
@@ -1,9 +1,8 @@
 # Template file for 'font-libertine-ttf'
 pkgname=font-libertine-ttf
 version=5.3.0
-revision=1
+revision=5
```
Comment (fosslinux) [inline srcpkgs/font-libertine-ttf/template:4]: =2

**Comment by Chocimier** [comment]
All builds. font-spleen fails do_check, please overwrite.

**Comment by fosslinux** [comment]
Could someone take a look at this again?

**Pull request closed without merging**